components/grid.css

```diff
@@ -1,11 +1,11 @@
 .grid {
   display: grid;
+  min-height: 100px;
```

Comment:
px to rem, maybe you forgot to change it.

Reply:
No, I just added it for the styling purpose, to show the demo.

Reply:
Ohh, then it's all cool. Merge it.